lib/fetch/sanitize.js

```diff
@@ -125,6 +125,17 @@ export function sanitizeTicket(json) {
   };
 }
 
+const categories = {
+  Leadership: 'leadership',
```

Comment:
Yes we don't really want the content editors to be able to misspell a tag and end up rendering that on the website. The only other option here is to manage the categories within Prismic but we couldn't find a good way to model it (whilst still allowing one person to be in multiple categories)